src/plugins/embeddable/public/lib/triggers/triggers.ts

```diff
@@ -53,31 +54,49 @@ export type ChartActionContext<T extends IEmbeddable = IEmbeddable> =
   | ValueClickContext<T>
   | RangeSelectContext<T>;
 
-export const isValueClickTriggerContext = (
-  context: ChartActionContext
-): context is ValueClickContext => context.data && 'data' in context.data;
-
-export const isRangeSelectTriggerContext = (
-  context: ChartActionContext
-): context is RangeSelectContext => context.data && 'range' in context.data;
-
 export const CONTEXT_MENU_TRIGGER = 'CONTEXT_MENU_TRIGGER';
 export const contextMenuTrigger: Trigger<'CONTEXT_MENU_TRIGGER'> = {
   id: CONTEXT_MENU_TRIGGER,
-  title: 'Context menu',
-  description: 'Triggered on top-right corner context-menu select.',
+  title: i18n.translate('embeddableApi.contextMenuTrigger.title', {
+    defaultMessage: 'Context menu',
+  }),
+  description: i18n.translate('embeddableApi.contextMenuTrigger.description', {
+    defaultMessage: 'A panel top-right corner context menu click.',
```

Comment:
I'd say something like: "Adds a new option to panel's context menu"

Reply:
Also noticed "Show option on... context menu" doesn't make much sense :( 
not sure what to do about it

![Screenshot 2020-10-29 at 10 40 22](https://user-images.githubusercontent.com/7784120/97551363-31d88780-19d3-11eb-8cbc-a17a8620f04f.png)